myhpi/settings.py

```diff
@@ -208,3 +209,26 @@
     constants.WARNING: "alert-warning",
     constants.ERROR: "alert-danger",
 }
+
+ENABLE_MAILING_LISTS = env.str("ENABLE_MAILING_LISTS")
```

Comment:
this could default to false IMO

Reply:
It is already false in the .env.example (I will change it to env.bool, since this is a mistake). What do you mean with defaulting to false?

Reply:
You can write `env.bool("ENABLE_MAILING_LISTS", False)` which will prevent django from throwing an IncorrectlyConfiguredError if the environment variable is not set